app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+import os
+
+
+def check_if_mv(command: str) -> None:
+    if command != "mv":
+        raise ValueError("Command must begging with 'mv' without flags")
+
+
+def create_dirs_from_str(destination: str) -> None:
+    if "/" in destination:
+        path = os.path.split(destination)[0]
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+
+def relocate_file_if_path_exists(source_file: str, destination: str) -> None:
+    if os.path.isdir(destination):
```

Comment:
if os.path.isdir(destination):
It is not good. the destination is dir than how you can move file?

Reply:
You solution is okay, but in the task we do not have this case

Reply:
I just thought about different possible cases. Can I leave it, or better to remove it?